# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295796415 28800
#      Sun Jan 23 07:26:55 2011 -0800
# Branch release-3-4-x
# Node ID 16b050c1a25ccb42cd8630f62f82462fda269b6a
# Parent  dfe958c044a26e703baf2f20b69249f76bbac5fd
Refer users to bug tracker, rather than octave-dev mailing list.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-22  Rik  <octave@nomad.inbox5.com>
+
+	* miscellaneous/info.m: Use reference to bug tracker rather than
+	octave-dev mailing list.
+
 2011-01-22  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/meshc.m: Add note: gnuplot does not support filled 3D patches,
 	or mixing non-filled contours with filled surfaces.
 	* plot/subplot.m: Set "box" to "on" by default (same as Matlab).
 	* plot/cylinder.m: Remove console output from demo.
 
 2011-01-22  Konstantinos Poulios  <logari81@googlemail.com>
diff --git a/scripts/miscellaneous/info.m b/scripts/miscellaneous/info.m
--- a/scripts/miscellaneous/info.m
+++ b/scripts/miscellaneous/info.m
@@ -32,13 +32,14 @@ function info ()
 \n\
   You may also find some information in the Octave Wiki at\n\
   http://wiki.octave.org\n\
 \n\
   Additional functionality can be enabled by using packages from\n\
   the Octave Forge project, which may be found at\n\
   http://octave.sourceforge.net\n\
 \n\
-  Report bugs to <bug@octave.org> (but first, please read\n\
-  http://www.octave.org/bugs.html to learn how to write a helpful report)\n\
-\n");
+  Report bugs to the bug tracker at\n\
+  http://bugs.octave.org\n\
+  But first, please read the guidelines to writing a helpful report at\n\
+  http://www.octave.org/bugs.html\n");
 
 endfunction
